### PR TITLE
ci: check agent on SEV/TDX runners

### PR DESCRIPTION
**Pull request by Manuthor**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# alias runner-sgx="ssh cosmian@162.19.87.37" # OVH
+# alias runner-sev="ssh cosmian@34.90.22.143" # GCP
+# alias runner-tdx="ssh cosmian@34.70.122.49" # GCP
+
```
Comment (ThibsG) [inline .github/workflows/ci.yml:4]: leftover?

File: README.md [hunk 2]
```diff
@@ -244,7 +246,7 @@ app_storage = "data/app"
 
 In that example, [`cosmian_helloworld`](https://github.com/Cosmian/helloworld-service) is the name of the application (as a `supervisor` service).
 
-The field `app_storage` defined the directory containing the configuration data of your application or any data used by the application. It is recommended to store it inside the Cosmian VM encrypted folder: `/var/lib/cosmian_vm/data`.
+The field `app_storage` defined the directory containing the configuration data of your application or any data used by the application. It is recommanded to store it inside the Cosmian VM encrypted folder: `/var/lib/cosmian_vm/data`.
```
Comment (ThibsG) [inline README.md:249]:
```suggestion
The field `app_storage` defined the directory containing the configuration data of your application or any data used by the application. It is recommended to store it inside the Cosmian VM encrypted folder: `/var/lib/cosmian_vm/data`.
```

**Review by ThibsG — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Manuthor) [reply .github/workflows/ci.yml:4]: indeed

File: README.md (hunk 2 above)
Comment (Manuthor) [reply README.md]: thanks

File: README.md [hunk 3]
```diff
@@ -99,7 +101,7 @@ This image:
 
 This is a abstract of the updated file tree:
 
-```
+```c
```
[the remark this answers is not in the document]
Comment (Manuthor) [reply README.md]: thanks

**Pull request merged**